brands/shop/supermarket.json

```diff
@@ -1590,6 +1590,18 @@
       "shop": "supermarket"
     }
   },
+  "shop/supermarket|GBarbosa": {
+    "count": 316,
+    "countryCodes": ["br"],
+    "tags": {
+      "brand": "GBarbosa Cencosud",
```

Comment:
I am guessing that "GBarbosa Cencosud" is name of company, not name used for its brand - right?

I suspect that

`       "brand": "GBarbosa",`

would be better (note that I have never seen GBarbosa shop in my life, that is just a guess)

Reply:
They have the word "Cencosud" on the logo after the name, but they don't mention it on advertising, texts... You can remove it, if you think it's more correct. I don't know well about the brand key.

Reply:
I made these changes and merged it

I also switched `website` -> `brand:website`.  I don't normally add these, since they can be fetched from Wikidata, but if people want to add them it doesn't break anything.